schema/layer.json

```diff
@@ -0,0 +1,15 @@
+{
+  "id": "/layer",
+  "type": "object",
+  "properties": {
+    "layer": { "type": "string" },
+    "count": { "type": "number" },
+    "geometry": { "type": "string" },
```

Comment:
Great, let's leave it like this - @ericfischer are these three geometries what you're working with as well? 

*note: the capital letters!*